spec/features/language_helper_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe 'receive_correct_diapers_form' do
+  context 'float numbers' do
+    it 'returns correct word form for float number' do
+      helper = LanguageHelper::UkrLanguageHelper.new
+    expect(helper.corr_word_form(421.0)).to eql('підгузок')
+    expect(helper.corr_word_form(34711.15)).to eql('підгузків')
+    expect(helper.corr_word_form(4532.10)).to eql('підгузки')
+    expect(helper.corr_word_form(32413.45)).to eql('підгузків')
+    expect(helper.corr_word_form(5634.23)).to eql('підгузки')
+    end
+    context 'integer numbers' do
+    it 'returns correct form for integer number' do
```

Comment:
every test should have a single expectation https://www.betterspecs.org/

Reply:
done